airflow/providers/docker/provider.yaml

```diff
@@ -63,7 +63,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.6.0
-  - docker>=5.0.3
+  - docker>=6
```

Comment:
I guess it also good idea to bump minimum version of the docker. Docker 6 released in August 2022